products/BellHybrid/services/time/tests/test-BellAlarmOperations.cpp

```diff
@@ -105,14 +105,16 @@ namespace
         std::unique_ptr<alarms::PreWakeUpSettingsProvider> &&preWakeUpSettingsProvider,
         std::unique_ptr<alarms::SnoozeChimeSettingsProvider> &&snoozeChimeSettingsProvider,
         std::unique_ptr<alarms::OnboardingSettingsProvider> &&onboardingSettingsProvider,
-        std::unique_ptr<alarms::AbstractBedtimeSettingsProvider> &&bedtimeSettingsProvider)
+        std::unique_ptr<alarms::AbstractBedtimeSettingsProvider> &&bedtimeSettingsProvider,
+        alarms::PreWakeUp::ChangeStateCallback callback = [](bool) {})
```

Comment:
You can just default this callback to `nullptr`, as far as I can see there's a check for null pointer before the function is called in AlarmOperations